README.md

```diff
@@ -9,7 +8,7 @@ The purpose of the installer is to place Ryujinx inside ~/.local/share/Ryujinx,
 
 Usage:
 Open a terminal, paste <br>
-`bash -c "$(curl -s https://raw.githubusercontent.com/edisionnano/Pine-jinx/main/pinejinx.sh)"` <br>
+`bash -c "$(curl -s https://raw.githubusercontent.com/essasetic/Pine-jinx/main/pinejinx.sh)"` <br>
```

Comment:
Might want to change this back to the original repo for the PR

Reply:
Ah yeah, oversight on my end.